src/app/src/main/kotlin/org/vaccineimpact/api/app/repositories/jooq/JooqUserRepository.kt

```diff
@@ -61,6 +71,16 @@ class JooqUserRepository : JooqRepository(), UserRepository
         )
     }
 
+    private fun getUser(username: String): AppUserRecord
```

Comment:
On the whole (for the sake of code readability) I prefer not to use ternary operators except for variable assignment